tests/upgrades/test_errata.py

```diff
@@ -249,11 +173,16 @@ def test_pre_scenario_generate_errata_for_client(self):
             delay=2,
             logger=self.logger
         )
-        self._install_or_update_package(client_container_id, 'katello-agent')
-        self._run_goferd(client_container_id)
+
+        CommonUpgradeUtility(container_id=client_container_id, package="katello-agent"). \
+            install_or_update_package(update=True)
+        self.assertIn("goferd", CommonUpgradeUtility(container_id=client_container_id).
```

Comment:
I think it would be great to keep assertion in utils itself .. else we have to everywhere specify this assertion everywhere when calling ..

Reply:
Fixed